api_tests/accounts_patch_success.yaml

```diff
@@ -0,0 +1,42 @@
+# Test generated by RoostGPT for test credential-wallet-api using AI Type Open AI and AI Model gpt-4
+
+
+# Test generated for /accounts_patch for http method type PATCH in artillery framework
+
+# RoostTestHash=2758affb7e
+
+
+config:
+  target: "{{ $processEnvironment.API_HOST }}"
+  plugins:
+    expect: {}
+  payload:
+    path: "accounts_patch_success.csv"
+    fields:
+      - "email"
+    skipHeader: true
+
+scenarios:
+  - name: "Successful API requests"
+    flow:
+      - post:
+          url: "/accounts"
+          headers:
+            Authorization: "{{ $processEnvironment.AUTH_TOKEN }}"
+          json:
+            email: "{{ email }}"
+          expect:
+            - statusCode: 200
+            - hasProperty: "headers.api-version"
+            - bodySchema:
+                type: "object"
+                properties:
+                  email:
+                    type: "string"
+                    format: "email"
+                    pattern: "^(([^<>()[\\]\\\\.,;:\\s@\\\"]+(\\.[^<>()[\\]\\\\.,;:\\s@\\\"]+)*)|(\\\".+\\\"))@((\\[[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}\\])|(([a-zA-Z\\-0-9]+\\.)+[a-zA-Z]{2,}))$"
+                    nullable: false
+          capture:
+            json: "$.email"
+            as: "email"
```

Comment:
```suggestion
```

body schema is not expected as per API spec. so capturing $.email from response is failing.

```
➜  api_tests (roost-1701745564) ✔ DEBUG=http:request,http:response,all artillery run  --target http://127.0.0.1:4011 accounts_patch_success.yaml
Test run id: tr47d_3paz4q7mbgw6ncejhqmytqfmpxqgq_8f79
Phase started: unnamed (index: 0, duration: 1s) 11:28:33(+0530)

Phase completed: unnamed (index: 0, duration: 1s) 11:28:34(+0530)

⠧ 2023-12-05T05:58:34.754Z http:request request start: /accounts
2023-12-05T05:58:34.766Z http:response {
  "access-control-allow-origin": "*",
  "access-control-allow-headers": "*",
  "access-control-allow-credentials": "true",
  "access-control-expose-headers": "*",
  "api-version": "1.0.0",
  "commit-hash": "928d28d",
  "content-type": "text/plain",
  "date": "Tue, 05 Dec 2023 05:58:34 GMT",
  "connection": "keep-alive",
  "keep-alive": "timeout=5",
  "content-length": "0"
}
worker error, id: 1 TypeError [Error]: Cannot read properties of undefined (reading 'ok')
    at /Users/mayankgaikwad/.nvm/versions/node/v18.12.1/lib/node_modules/artillery/node_modules/artillery-plugin-expect/index.js:125:11
    at Array.forEach (<anonymous>)
    at expectationsPluginCheckExpectations (/Users/mayankgaikwad/.nvm/versions/node/v18.12.1/lib/node_modules/artillery/node_modules/artillery-plugin-expect/index.js:124:31)
    at iteratee (/Users/mayankgaikwad/.nvm/versions/node/v18.12.1/lib/node_modules/artillery/node_modules/@artilleryio/int-core/lib/engine_http.js:600:19)
    at /Users/mayankgaikwad/.nvm/versions/node/v18.12.1/lib/node_modules/artillery/node_modules/async/dist/async.js:3113:16
    at replenish (/Users/mayankgaikwad/.nvm/versions/node/v18.12.1/lib/node_modules/artillery/node_modules/async/dist/async.js:1014:17)
    at /Users/mayankgaikwad/.nvm/versions/node/v18.12.1/lib/node_modules/artillery/node_modules/async/dist/async.js:1019:9
    at eachLimit$1 (/Users/mayankgaikwad/.nvm/versions/node/v18.12.1/lib/node_modules/artillery/node_modules/async/dist/async.js:3199:24)
    at Object.<anonymous> (/Users/mayankgaikwad/.nvm/versions/node/v18.12.1/lib/node_modules/artillery/node_modules/async/dist/async.js:1049:16)
    at captured (/Users/mayankgaikwad/.nvm/versions/node/v18.12.1/lib/node_modules/artillery/node_modules/@artilleryio/int-core/lib/engine_http.js:582:21)
➜  api_tests (roost-1701745564) ✔
```